frontend/src/pages/Configure/Storage.js

```diff
@@ -542,11 +533,12 @@ function StorageInstance({index}) {
                         value={existingId}
                         onChange={(({detail}) => {setState(existingPath, detail.value)})} />
                     </div>,
-                  "FsxLustre": <FormField label="FSx Filesystem">
+                  "FsxLustre": <FormField label="FSx Lustre Filesystem">
                     <Select
                       placeholder="Please Select"
                       selectedOption={existingId && idToOption(existingId)} label="FSx Filesystem" onChange={({detail}) => {setState(existingPath, detail.selectedOption.value)}}
-                      options={fsxFilesystems.map((x, i) => {return {value: x.FileSystemId, label: (x.FileSystemId + (fsxName(x) ? ` (${fsxName(x)})` : ""))}})}
+                      options={fsxFilesystems.lustre.map((fs) => {return {value: fs.id, label: fs.name}})}
```

Comment:
issue: As far as I can tell, we can have `fs.name === null` but we got rid of the `FileSystemID` as part of the label. Can't we end up in a situation where we have an empty label?

Reply:
Good point, I'll restore the original behavior, and I'll keep the same also for OPENZFS and ONTAP

Reply:
Fixed!